src/Features/CSharp/Portable/Completion/CompletionProviders/SymbolCompletionProvider.ItemRules.cs

```diff
@@ -48,6 +50,21 @@ public static string GetInsertionText(ISymbol symbol, AbstractSyntaxContext cont
 
                 return symbol.Name.EscapeIdentifier(isQueryContext: context.IsInQuery);
             }
+
+            public override bool? IsBetterPreselectedMatch(CompletionItem item, CompletionItem other, string textTypedSoFar)
+            {
+                var symbolItem = item as SymbolCompletionItem;
```

Comment:
Yes, this is only called if the provider and indicated priority matched--will doc this.